docker-compose.yml

```diff
@@ -148,8 +165,6 @@ services:
       - COURSIER_CACHE=/opt/raster-foundry/app-backend/.coursier-cache
       - RF_LOG_LEVEL=INFO
       - SBT_OPTS="-Xmx2G -XX:MaxPermSize=2G"
-    ports:
```

Comment:
This is going to cause `- "9030:9020"` to get tucked under `environment` vs. `ports`.